sample/todo-sync/src/com/cloudant/todo/TasksModel.java

```diff
@@ -72,7 +72,7 @@ public TasksModel(Context context) {
         );
         DatastoreManager manager = DatastoreManager.getInstance(path.getAbsolutePath());
         try {
-            this.mDatastore = manager.openDatastore(TASKS_DATASTORE_NAME);
+            this.mCloudantSync = manager.openDatastore(TASKS_DATASTORE_NAME);
```

Comment:
@rhyshort yes this also exposes some other naming issues. `DatastoreManager` probably isn't an appropriate name any more.